src/elements/Input/Input.js

```diff
@@ -135,6 +142,23 @@ class Input extends Component {
 
   handleInputRef = c => (this.inputRef = c)
 
+  partitionProps = () => {
+    const { disabled, onChange, type } = this.props
+
+    const tabIndex = this.computeTabIndex()
+    const unhandled = getUnhandledProps(Input, this.props)
+    const [htmlInputProps, rest] = partitionHTMLInputProps(unhandled)
+
+    htmlInputProps.ref = this.handleInputRef
+    htmlInputProps.type = type
+
+    if (disabled) htmlInputProps.disabled = disabled
+    if (onChange) htmlInputProps.onChange = this.handleChange
+    if (tabIndex) htmlInputProps.tabIndex = tabIndex
```

Comment:
The problem is that the component receives `disabled` as `undefined`, same is for `tabIndex`

I made tests working, however, it's not the best idea. May be better solution possible there?